File: flaskr/bin/logger.py
def write_to_file(file_name:str, text:str):
    with open("out/"+file_name, "w") as f:
        f.write(text)

def append_to_file(file_name:str, text:str):
    with open("out/"+file_name, "a") as f:
        f.write(text)

def log(text:str):
    append_to_file("log.txt", text + "\n")

File: server/bin/logger.js
const fs = require('fs');

function writeToFile(file_name, text) {
    fs.writeFile(`log/${file_name}`, text, (err) => {
        if (err) throw err;
        console.log('The file has been saved!');
    });
}

function appendToFile(file_name, text) {
    fs.appendFile(`log/${file_name}`, text, (err) => {
        if (err) throw err;
        console.log('The data was appended to file!');
    });
}

function log(text) {
    appendToFile("log.txt", text + "\n");
}

module.exports = { log };